pkg/manifests/manifests.go

```diff
@@ -2740,6 +2741,10 @@ func (f *Factory) NewValidatingWebhook(manifest io.Reader) (*admissionv1.Validat
 	return NewValidatingWebhook(manifest)
 }
 
+func (f *Factory) ThanosQuerierPodDisruptionBudget() (*policyv1.PodDisruptionBudget, error) {
+	return f.NewPodDisruptionBudget(f.assets.MustNewAssetReader(ThanosQuerierPodDisruptionBudget))
```

Comment:
we also need to make sure the PDB namespace is the same as the one from the factory. You can look at https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/manifests/manifests.go#L1622-L1633 for an example.